index.js

```diff
@@ -8,14 +8,24 @@ const clone = require('lodash.clonedeep');
 const uuid = require('uuid/v4');
 
 /**
+ * Proxy polyfill injection.
+ */
+
+require('proxy-polyfill');
```

Comment:
As discussed offline, we will be pursuing a `vm.runInContext` experiment and, as a last resort, bundling a modified version of the Proxy module that returns the polyfill instead of mutating the global namespace.

If anyone wants to change that polyfill implementation, they'll just have to expose it under `global.Proxy`, at which points this module will no longer load it's own.